src/sentry/api/endpoints/api_tokens.py

```diff
@@ -83,15 +87,23 @@ def delete(self, request: Request):
         if is_active_superuser(request):
             user_id = request.data.get("userId", user_id)
         # TODO: we should not be requiring full token value in the delete endpoint, and should instead be using the id
-        token = request.data.get("token")
-        if not token:
-            return Response({"token": ""}, status=400)
+        token = request.data.get("token", None)
+        token_id = request.data.get("tokenId", None)
+        # Account for token_id being 0, which can be considered valid
```

Comment:
:squint:

huh. I guess... can we just assume it's not valid and not account for this edge case? 🤔 
since we generate the tokens, i don't think that should be an issue right?

Reply:
I'm just trying to keep self hosted folks in mind haha. If they decided to start or reset index at 0 for auto increment, technically it's a valid token so I don't want their first tokens to have a weird experience.